concourse/steps/release.py

```diff
@@ -349,6 +353,8 @@ def _add_all_and_create_commit(self, message: str):
         commit = self.git_helper.index_to_commit(
             message=message,
         )
+        if not commit:
```

Comment:
by splitting this step into multiple steps, this early-exit will no longer be required